tetris.py

```diff
@@ -104,6 +106,16 @@ def right_mediator(self, event):
         self.draw_block()
 
 
+color_chart = {
+    1: "turquoise",
+    2: "yellow green",
+    3: "yellow",
+    4: "gold",
+    5: "orange",
+    6: "dark orange",
+    7: "red2",
+    8: "red4",
```

Comment:
Maybe the keys should be tetris block shape names, e.g. `"L_rev": "red2"` instead of `7: "red2"`